main.go

```diff
@@ -50,6 +45,8 @@ const (
 	TNFReportKey                  = "cnf-certification-test"
 	extraInfoKey                  = "testsExtraInfo"
 	noLabelsExpr                  = "none"
+	logFileName                   = "cnf-test-log"
```

Comment:
I know it's just a sample file name, but I believe it should have an extension. My proposal for a default name is `cnf-certsuite.log`.

Reply:
Done.